app/javascript/src/NewService/components/FormElements/ServiceManualListItems.jsx

```diff
@@ -1,36 +1,51 @@
 // @flow
 
-import React from 'react'
+import React, {useState} from 'react'
 import {Label} from 'NewService/components/FormElements'
+import type {ServiceFormTemplate} from 'NewService/types'
 
-const ServiceManualListItems = () =>
-  <React.Fragment>
-    <li id="service_name_input" className="string required">
-      <Label
-        htmlFor='service_name'
-        label='Name'
-        required
-      />
-      <input maxLength="255" id="service_name" type="text" name="service[name]" autoFocus="autoFocus"/>
-    </li>
-    <li id="service_system_name_input" className="string required">
-      <Label
-        htmlFor='service_system_name'
-        label='System name'
-        required
-      />
-      <input maxLength="255" id="service_system_name" type="text" name="service[system_name]"/>
-      <p className="inline-hints">
-        Only ASCII letters, numbers, dashes and underscores are allowed.
-      </p>
-    </li>
-    <li id="service_description_input" className="text optional">
-      <Label
-        htmlFor='service_description'
-        label='Description'
-      />
-      <textarea rows="3" id="service_description" name="service[description]"></textarea>
-    </li>
-  </React.Fragment>
+type Props = ServiceFormTemplate
 
+const ServiceManualListItems = ({service, errors}: Props) => {
+  const [name, setName] = useState(service.name)
+  const [systemName, setSystemName] = useState(service.system_name)
+  const [description, setDescription] = useState(service.description)
+  const onChange = (fn) => (e) => fn(e.currentTarget.value)
+
+  return (
+    <React.Fragment>
+      { /* $FlowFixMe boolean coercion in string literals */ }
+      <li id="service_name_input" className={`string required ${!!errors.name && 'error'}`}>
```

Comment:
There's two problems that I just realized, tho:
* `errors.name` is a maybe type, potentially throwing an error. This is obfuscated by the FlowFixMe comment.
* When `name` is falsy, the string will be `string required false`. It has to be `${!!errors.name ? 'error' : ''}`

Reply:
```suggestion
      <li id="service_name_input" className={`string required ${errors && errors.name ? 'error' : ''}`}>
```

Reply:
* `errors.name` could be `undefined`, but not `null`, so won't be throwing any exceptions.
* And yes! nice catch, it would be the wrong string ("false")

Reply:
You're right, I was thinking about `errors` being undefined, but that can't happen.